packages/components/alert/src/components/alert-description/AlertDescription.tsx

```diff
@@ -0,0 +1,37 @@
+import { Typography } from '@julo-ui/typography';
+import { forwardRef, cx } from '@julo-ui/system';
+
+import { useAlertContext } from '../../AlertProvider';
+
+import { alertDescription } from './styles';
+import type { AlertDescriptionProps } from './types';
+
+const AlertDescription = forwardRef<AlertDescriptionProps, 'div'>(
+  (props, ref) => {
+    const { children, className, css, sx, ...resProps } = props;
+    const alertContext = useAlertContext();
+
+    if (!alertContext) throw new Error('AlertIcon should be within Alert');
+
+    const { status } = alertContext;
+
+    return (
+      <Typography
+        bold
+        as='div'
+        type='caption'
+        ref={ref}
+        sx={{ ...alertDescription[status], ...sx }}
+        css={css}
```

Comment:
it's unnecessary to pass `css` here cause you don't do anything that need to prevent the value from being override,  you can just simple destruct `resProps` and it'll pass the css.